onnxruntime/test/providers/cpu/math/matmul_integer_test.cc

```diff
@@ -30,5 +30,54 @@ TEST(MatmulIntegerOpTest, MatMulInteger) {
   test.AddOutput<int32_t>("T3", {1, 1}, {-1});
   test.Run();
 }
+
+TEST(MatmulIntegerOpTest, MatMulInteger_Uint8_Int8_GEMM) {
+  OpTester test("MatMulInteger", 10);
+  test.AddInput<uint8_t>("T1", {2, 40}, {4, 26, 2, 69, 76, 14, 14, 44, 58, 2, 21, 21, 60, 32, 59, 23, 74, 56, 37, 25, 26, 31, 60, 37, 46, 18, 30, 56, 3, 38, 55, 72, 31, 56, 37, 52, 11, 50, 57, 74, 48, 59, 33, 0, 51, 22, 73, 50, 70, 43, 34, 63, 74, 63, 34, 15, 43, 36, 52, 72, 7, 17, 33, 49, 3, 67, 47, 12, 75, 62, 43, 46, 43, 3, 18, 8, 61, 19, 58, 13});
+  test.AddInput<int8_t>("T2", {40, 2}, {12, 10, 9, 13, 2, 7, 10, 4, 17, 15, 14, 19, 4, 11, 1, 19, 7, 0, 12, 3, 13, 8, 4, 12, 16, 18, 15, 12, 9, 8, 12, 19, 14, 11, 18, 15, 9, 3, 0, 13, 5, 1, 5, 3, 8, 3, 2, 17, 0, 0, 4, 13, 6, 12, 11, 17, 9, 10, 5, 8, 15, 5, 4, 18, 5, 0, 4, 10, 11, 4, 4, 0, 9, 14, 14, 17, 19, 13, 3, 13});
+  test.AddOutput<int32_t>("T3", {2, 2}, {14110, 15747, 14114, 17441});
```

Comment:
Would have expected to see max and min uint8_t and int8_t values in the test input for all these tests so edge cases are tested. 

e.g. calculations involving max uint8_t * min int8_t, max uint8_t * max int8_t, etc. etc.